src/pages/contact_information.tsx

```diff
@@ -99,13 +87,14 @@ const ContactCard = ({ title, operationTime, operationTitle, imgUrl, email }: Co
 				<Button component="a" href={`mailto:${email}`} className="h-10 w-full">
 					Email
 				</Button>
-				<Button 
-					onClick={handleCopyToClipboard}
-					className={`mt-2 h-10 w-full ${isCopied ? 'bg-green-500' : ''}`}
-					variant={isCopied ? 'filled' : 'light'}
-				>
-					{isCopied ? 'Copied!' : 'Copy Email Address'}
-				</Button>
+
+				<CopyButton value={email}>
+					{({ copied, copy }) => (
+						<Button color={copied ? "teal" : "blue"} onClick={copy} className="h-10 w-full mt-4">
```

Comment:
brother what happened to the `variant={isCopied ? 'filled': 'light'}` 

`color={copied ? "teal" : "blue"}` doesn't work on my device btw
<img width="298" alt="image" src="https://github.com/stamford-syntax-club/stamfordcenter-frontend/assets/92321280/10294482-3194-42e7-b01e-2a7d87fb75d7">

Reply:
uhh, i seee, i willl chanage it back then.

Reply:
is the margin  too far apart or it is just fine? and i'll fix the color